mix.exs

```diff
@@ -32,7 +32,8 @@ defmodule RedisMutex.Mixfile do
 
       # Dev and test dependencies
       {:credo, "~> 1.7", only: [:dev, :test]},
```

Comment:
since you are adding typespec, we should add dialyzer and make sure it s happy.

Reply:
Thanks for the suggestion. Done